tests/model_selection/test_hyperband.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+import scipy.stats as stats
+import math
+
+from distributed import Client
+from distributed.utils_test import loop, cluster
+
+from dask_ml.datasets import make_classification
+from dask_ml.model_selection import HyperbandCV
+
+
+class ConstantFunction:
+    def _fn(self):
+        return self.value
```

Comment:
Hmm, should the `__init__` take a `value` argument? Otherwise (I think) this is undefined.